showcase/package.json

```diff
@@ -56,16 +56,16 @@
     "@typescript-eslint/eslint-plugin": "^8.5.0",
     "@typescript-eslint/parser": "^8.5.0",
     "broccoli-asset-rev": "^3.0.0",
-    "concurrently": "^8.2.2",
-    "ember-a11y-testing": "^6.1.1",
+    "concurrently": "^9.1.0",
+    "ember-a11y-testing": "^7.0.2",
     "ember-auto-import": "^2.7.3",
     "ember-basic-dropdown": "^8.1.0",
     "ember-body-class": "^3.0.0",
-    "ember-cli": "~5.9.0",
+    "ember-cli": "~6.0.1",
```

Comment:
it's not a problem per se to try to do this here, but I think it'd be better to wait until we run the ember-cli 6.0 update blueprint (same comment for the website as well).